src/browser/modules/Sidebar/Sidebar.tsx

```diff
@@ -19,24 +19,41 @@
  */
 
 import React, { ReactFragment, ReactElement } from 'react'
+import uuid from 'uuid'
 import { connect } from 'react-redux'
 import DatabaseDrawer from '../DBMSInfo/DBMSInfo'
 import DocumentsDrawer from './Documents'
 import AboutDrawer from './About'
 import SettingsDrawer from './Settings'
 import Favorites from './favorites'
 import ProjectFilesDrawer from './ProjectFiles'
-import StaticScripts from './static-scripts'
 import TabNavigation from 'browser-components/TabNavigation/Navigation'
 import { DrawerHeader } from 'browser-components/drawer'
+import NewSavedScript from './NewSavedScript'
 import BrowserSync from '../Sync/BrowserSync'
 import { isUserSignedIn } from 'shared/modules/sync/syncDuck'
+import { addFavorite } from 'shared/modules/favorites/favoritesDuck'
 import { useBrowserSync } from 'shared/modules/features/featuresDuck'
 import {
   PENDING_STATE,
   CONNECTED_STATE,
   DISCONNECTED_STATE
 } from 'shared/modules/connections/connectionsDuck'
+import {
+  getCurrentDraft,
+  setDraftScript
+} from 'shared/modules/sidebar/sidebarDuck'
+import { isRelateAvailable } from 'shared/modules/app/appDuck'
+
+function favoriteName(cmd: string) {
```

Comment:
worth adding some comment here about this, as we have similar stuff in the project file utils

Reply:
I'll move it there!

Reply:
Done

Reply:
move it to project file utils? Or have you combined the utils now?

Reply:
I wouldnt put this in project file utils, unless you rename the file to be more generic. Or maybe just leave a comment here instead? What do you think?

Reply:
Tried to avoid adding any mention of "favorites" in those utils, and its Project File focused